src/situations/commun/vues/cadre.js

```diff
@@ -69,7 +81,7 @@ export default class VueCadre {
 
   previensLaFermetureDeLaSituation ($) {
     $(window).on('beforeunload', (e) => {
-      if (![CHARGEMENT, ERREUR_CHARGEMENT, ATTENTE_DEMARRAGE, FINI, STOPPEE].includes(this.situation.etat())) {
+      if ([ENTRAINEMENT_DEMARRE, ENTRAINEMENT_FINI, DEMARRE].includes(this.situation.etat())) {
```

Comment:
pourquoi on retire `STOPPEE` ?

Reply:
J'ai changé la condition pour pas que ce soit un `not`.

Reply:
oui merci !